combat-trainer.lic

```diff
@@ -1883,8 +1902,8 @@ class ManipulateProcess
       index = npc_occurrences[npc]
       ordinal_string = $ORDINALS[index]
 
-      case bput("manipulate friendship #{ordinal_string} #{npc}", 'You\'re already manipulating', 'beyond your ken', 'You attempt to empathically manipulate', 'You strain', 'does not seem to have a life essence')
-      when 'does not seem to have a life essence' then
+      case bput("manipulate friendship #{ordinal_string} #{npc}", 'You\'re already manipulating', 'beyond your ken', 'You strain', 'does not seem to have a life essence', 'is definitely attracted to you', 'Roundtime', 'You have certainly attracted', 'Manipulate what\?')
+      when 'does not seem to have a life essence', 'is definitely attracted to you', 'You have certainly attracted' then
```

Comment:
The old code had this, but I don't think the `then` is required here.

Reply:
removed the then
testing now